rb/selenium-devtools.gemspec

```diff
@@ -41,5 +41,5 @@ Gem::Specification.new do |s|
 
   s.require_paths = ['lib']
 
-  s.add_runtime_dependency 'selenium-webdriver', '~> 4.2'
+  s.add_runtime_dependency 'selenium-webdriver', '~> 4.2.0'
```

Comment:
This needs to be reverted

Reply:
> This needs to be reverted

I reverted this on my latest commit, thank you for spotting it

I also tested locally with jruby-9.3.9.0 on my M1 which versions of Steep and RBS worked and which one was the last stable so it's 1.2.0 and 0.46.0